mojaloop/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: Mojaloop Helm chart for Kubernetes
 name: mojaloop
-version: 15.1.0
-appVersion: "ml-api-adapter: v14.0.0; central-ledger: v17.0.3; account-lookup-service: v14.1.0; quoting-service: v15.0.2; central-settlement: v15.0.0; bulk-api-adapter: v15.0.3; transaction-requests-service: v14.1.0; simulator: v12.0.0; mojaloop-simulator: v14.0.1; sdk-scheme-adapter: v23.0.1; auth-service: v14.0.0; consent-oracle: v0.2.0; thirdparty-sdk: v13.0.2; ml-testing-toolkit: v16.1.1; ml-testing-toolkit-ui: v15.4.0;"
+version: 15.2.0
+appVersion: "ml-api-adapter: v14.0.4; central-ledger: v17.3.0; account-lookup-service: v14.2.5; quoting-service: v15.2.2; central-settlement: v15.0.3; bulk-api-adapter: v16.0.0; transaction-requests-service: v14.1.1; simulator: v12.0.0; mojaloop-simulator: v15.0.0; sdk-scheme-adapter: v23.1.1; auth-service: v14.0.1; consent-oracle: v0.2.1; thirdparty-sdk: v15.1.1; ml-testing-toolkit: v17.0.0; ml-testing-toolkit-ui: v15.4.0;"
```

Comment:
@oderayi take note that the `ml-testing-toolkit-ui` has not been bumped?

Reply:
Yes @mdebarros, there was no update to ttk-ui in this release